packages/core/components/HeaderRibbon/tutorials/FilterFiles.tsx

```diff
@@ -0,0 +1,28 @@
+import { Icon } from "@fluentui/react";
+import * as React from "react";
+
+import Tutorial from "../../../entity/Tutorial";
+
+export const FILTER_FILES_TUTORIAL = new Tutorial("Filtering")
+    .addStep({
+        targetId: Tutorial.ANNOTATION_LIST_ID,
+        message: (
+            <span>
+                Filters for <strong>annotations</strong> can be found and used by clicking the{" "}
+                <Icon iconName="FilterSolid" /> icon for the annotation to filter by and selecting
+                which annotation values the files must have
+            </span>
+        ),
+    })
+    .addStep({
+        targetId: Tutorial.FILE_ATTRIBUTE_FILTER_ID,
+        message:
+            "Files can also be filtered by these attributes by selecting one from the dropdown menu (ex. 'Uploaded') " +
+            'and entering a value. The value for attributes like "File "name" do not have to be exact and can instead be ' +
+            'partial matches (ex. entering "AD0000057" would show file "AD00000573_100x_20220729_H01_001_Scene-44_aligned.ome.tiff")',
```

Comment:
I would mention in this step that all files have the attributes in the dropdown at the top (that's true, right? Or at least mostly true? IDK about thumbnail path). Maybe add another step between this one and the previous, attached to the attribute dropdown. It would say:

> All files have the attributes listed in this dropdown.

And then:

> You can filter on the selected attribute by entering a value here. These values do not have to be exact, e.g. entering a File Name of "ZSD1" would return all files with names starting with or containing "ZSD1".

Reply:
I dig it 👍